implementation.md

```diff
@@ -353,13 +403,13 @@ To create an action:
 1. Annotate this class with `@MetaInfServices`.
```

Comment:
The _type_ thing above confused me. I was expecting an annotation or something. Maybe provide an example